tutorial/index.jade

```diff
@@ -1,10 +1,155 @@
 .chapter(data-title='Register a User')
-  include chapter1
-.chapter(data-title='Send a One-Time Code')
-  include chapter2
-.chapter(data-title='Verify the Code')
-  include chapter3
-.chapter(data-title='Resend the Code')
-  include chapter4
-.chapter(data-title='Confirm Account Creation')
-  include chapter5
\ No newline at end of file
+  .step(data-title='Introduction')
+    :markdown
+      ## Introduction
+
+      Ready to implement user account verification in your application? Here's how it works at a high level:
+
+      1. The user begins the registration process by entering their data, including a phone number, into a signup form
+
+      1. The authentication system sends a one-time password to the user's mobile phone to verify their possession of that phone number
+
+      1. The user enters the one-time password into a form before completing registeration
+
+      1. The user sees a success page and receives an SMS indicating that their account has been created
+
+      ## Building Blocks
+
+      To get this done, you'll be working with the following Twilio-powered APIs:
+
+      **Authy REST API**
+
+      * [Users Resource](https://docs.authy.com/#section-Enabling_two-factor_on_a_user): You will need to create Authy users to send and verify one-time passwords with them.
```

Comment:
Nitpick: You should use **protocol relative URL**. Would you mind updating all the URLs in this tutorial to accordingly?